conda_build/post.py

```diff
@@ -396,7 +396,9 @@ def osx_ch_link(path, link_dict, host_prefix, build_prefix, files):
 
 
 def mk_relative_osx(path, host_prefix, build_prefix, files, rpaths=('lib',)):
-    assert sys.platform == 'darwin'
+    if sys.platform != 'darwin':
+      log.warn("Found Mach-O file but patching is only supported on macOS, skipping: %s", path)
+      return
```

Comment:
Is it a standard style (2 blank space indents) here in conda-build? I saw 4 space indents in many other codes of the code base.

Reply:
It should be 4 spaces. Can you please send a PR to fix it?

Reply:
PR #3925 adjusts the indentation and fixes another bug introduced here.

Reply:
Thank you!